lib/Basics/sdt.h

```diff
@@ -5,7 +5,7 @@
 */
 
 #ifndef _SYS_SDT_H
-#define _SYS_SDT_H    1
```

Comment:
Are changes in this file intentional?

Reply:
weird. yes. no idea, where these changes came from.